### PR TITLE
'big cleanup'

### PR DESCRIPTION
**Pull request by bioshazard**

### CODE REVIEW

File: lf/system/admin/index.php [hunk 1]
```diff
@@ -46,7 +45,7 @@
 
 // Generate new User() and test access
 $user = new User();
-
+$user->fromSession();
```
Comment (kripplek) [inline lf/system/admin/index.php:49]: $user=(new User)->fromSession();

File: lf/system/lib/user.php [hunk 2]
```diff
@@ -1,32 +1,53 @@
...
+		}
+		else if(is_string($details))
+		{
+			$this->setDetails(
+				orm::qUsers('lf')
+					->cols('id, access, user, display_name')
+					->byDisplay_name($details)
+					->first()
+			);
+		}
```
Comment (kripplek) [inline lf/system/lib/user.php:40]: mmmmmm dats nice

**Comment by kripplek** [comment]
This looks good. I really like the way it's shaping up.

File: lf/system/admin/index.php (hunk 1 above)
Comment (bioshazard) [inline lf/system/admin/index.php:49]: I was deving on PHP 5.3 for the longest time. Finally upgraded today. Gonna move everything to that nicer oneliner format ;)

**Comment by bioshazard** [comment]
Yea, the user updates were a breakthrough yesterday.

Now you can get a user with

```
$user = new User('BIOS');
```

or 

```
$user = new User(39);
```

**Pull request merged**